packages/bootleg-tokens/contracts/ERC721/ERC721.sol

```diff
@@ -134,21 +134,21 @@ contract ERC721 is ERC165, IERC721 {
         _transferFrom(from, to, tokenId);
     }
 
-    /**
-     * @dev Safely transfers the ownership of a given token ID to another address
-     * If the target address is a contract, it must implement `onERC721Received`,
-     * which is called upon a safe transfer, and return the magic value
-     * `bytes4(keccak256("onERC721Received(address,address,uint256,bytes)"))`; otherwise,
-     * the transfer is reverted.
-     *
-     * Requires the msg sender to be the owner, approved, or operator
-     * @param from current owner of the token
-     * @param to address to receive the ownership of the given token ID
-     * @param tokenId uint256 ID of the token to be transferred
-    */
-    function safeTransferFrom(address from, address to, uint256 tokenId) public payable {
-        safeTransferFrom(from, to, tokenId, "");
-    }
+    // /**
```

Comment:
I'd prefer to not comment this out but actually solve the compiler problem. This is a quick fix. Open to any/other ideas here... FWIW the error was/is
```
$ truffle compile
Compiling ./contracts/BootlegTraderApp.sol...

/Users/brian/Projects/bootleg/packages/bootleg-app-contracts/contracts/BootlegTraderApp.sol:36:5: TypeError: Member "safeTransferFrom" not unique after argument-dependent lookup in contract BootlegToken.
    bootlegToken.safeTransferFrom.value(msg.value)(owner, newOwner, tokenId, "");
    ^---------------------------^
Compilation failed. See above.
Truffle v5.0.4 (core: 5.0.4)
Node v10.15.1
```

Reply:
Could you leave a comment as to why it's commented out in the code? Maybe with an issue opened against truffle or solc?

I'll pull this down and see if I can make some headway with it.

Reply:
will do!